core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -88,6 +90,10 @@ abstract class Chunk[+O] extends Serializable with ChunkPlatform[O] with ChunkRu
     Chunk.array(b.result()).asInstanceOf[Chunk[O2]]
   }
 
+  /** Returns true if the Chunk contains the given element. */
+  def contains[O2 >: O: Eq](elem: O2): Boolean =
+    iterator.exists(o => Eq[O2].eqv(o, elem))
```

Comment:
I will be calling the Eq's summoner for every `o`. Even though it should not have a huge impact, I feel it would be better to avoid it:
```suggestion
  def contains[O2 >: O](elem: O2)(implicit ev: Eq[O2]): Boolean =
    iterator.exists(o => ev.eqv(o, elem))
```